lib/modules/manager/fingerprint.spec.ts

```diff
@@ -0,0 +1,10 @@
+import { getManagers, hashMap } from '.';
+
+describe('modules/manager/fingerprint', () => {
+  it('validate manager hash', () => {
+    const managers = getManagers();
+    for (const [manager] of managers) {
+      expect(hashMap.get(manager)).toBeDefined();
```

Comment:
Maybe better check that it looks like a `sha256` so we don't accidently have emtpy strings?